2022-04-15 23:38:32 DEBUG PersonaDAO:22 - This is debug : Holaj
2022-04-15 23:38:32 INFO  PersonaDAO:26 - This is info : Holaj
2022-04-15 23:38:32 WARN  PersonaDAO:29 - This is warn : Holaj
2022-04-15 23:38:32 ERROR PersonaDAO:30 - This is error : Holaj
2022-04-15 23:38:32 FATAL PersonaDAO:31 - This is fatal : Holaj
2022-04-15 23:55:32 DEBUG PersonaDAO:22 - This is debug : Holaj
2022-04-15 23:55:32 INFO  PersonaDAO:26 - This is info : Holaj
2022-04-15 23:55:32 WARN  PersonaDAO:29 - This is warn : Holaj
2022-04-15 23:55:32 ERROR PersonaDAO:30 - This is error : Holaj
2022-04-15 23:55:32 FATAL PersonaDAO:31 - This is fatal : Holaj
2022-04-16 12:30:31 WARN  PersonaDAO:113 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Column 'telefono' not found.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:584)
	at com.mysql.cj.jdbc.result.ResultSetImpl.getString(ResultSetImpl.java:896)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.PersonaDAO.findPersonById(PersonaDAO.java:101)
	at test.uv.fei.tutorias/uv.fei.tutorias.PersonaDAOTest.findPersonById(PersonaDAOTest.java:19)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-16 13:43:02 WARN  PersonaDAO:174 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: No value specified for parameter 4
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1121)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.PersonaDAO.addPersonaReturnId(PersonaDAO.java:164)
	at test.uv.fei.tutorias/uv.fei.tutorias.PersonaDAOTest.addPersonReturnId(PersonaDAOTest.java:75)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-16 13:44:07 WARN  PersonaDAO:174 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: No value specified for parameter 6
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1121)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.PersonaDAO.addPersonaReturnId(PersonaDAO.java:164)
	at test.uv.fei.tutorias/uv.fei.tutorias.PersonaDAOTest.addPersonReturnId(PersonaDAOTest.java:75)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-16 22:57:57 WARN  PersonaDAO:50 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLSyntaxErrorException: Unknown column 'PE.nombre' in 'field list'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProfesorDAO.findProfesorById(ProfesorDao.java:44)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProfesorDaoTest.findProfesorById(ProfesorDaoTest.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 12:22:35 WARN  PersonaDAO:32 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() with statements that produce result sets
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1075)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProfesorDAO.addProfesor(ProfesorDao.java:28)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProfesorDaoTest.addProfesor(ProfesorDaoTest.java:17)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 12:26:12 WARN  PersonaDAO:32 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() with statements that produce result sets
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1075)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProfesorDAO.addProfesor(ProfesorDao.java:28)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProfesorDaoTest.addProfesor(ProfesorDaoTest.java:17)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 12:43:34 WARN  PersonaDAO:32 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Can not issue executeUpdate() or executeLargeUpdate() with statements that produce result sets
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1075)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProfesorDAO.addProfesor(ProfesorDao.java:28)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProfesorDaoTest.addProfesor(ProfesorDaoTest.java:17)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 12:51:16 WARN  PersonaDAO:50 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Profesor not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProfesorDAO.findProfesorById(ProfesorDao.java:46)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProfesorDaoTest.findProfesorById(ProfesorDaoTest.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 22:23:47 WARN  ExperienciaEducativa:35 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Experiencia educativa not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ExperienciaEducativaDAO.findExperienciasEducativasByName(ExperienciaEducativaDAO.java:27)
	at test.uv.fei.tutorias/uv.fei.tutorias.ExperienciaEducativaDAOTest.findExperienciasEducativasByName(ExperienciaEducativaDAOTest.java:27)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 23:10:53 INFO  ExperienciaEducativa:78 - Experiencia educativa eliminada con id2
2022-04-17 23:12:03 WARN  ExperienciaEducativa:81 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Error: No se ha eliminado ninguna experiencia educativa
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ExperienciaEducativaDAO.deleteExperienciaEducativa(ExperienciaEducativaDAO.java:76)
	at test.uv.fei.tutorias/uv.fei.tutorias.ExperienciaEducativaDAOTest.deleteExperienciaEducativa(ExperienciaEducativaDAOTest.java:49)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-17 23:39:21 WARN  ExperienciaEducativa:57 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLSyntaxErrorException: Table 'sistemadetutorias.experienciaeductiva' doesn't exist
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeQuery(ClientPreparedStatement.java:1009)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ExperienciaEducativaDAO.findExperienciaEducativaById(ExperienciaEducativaDAO.java:50)
	at test.uv.fei.tutorias/uv.fei.tutorias.ExperienciaEducativaDAOTest.findExperienciaEducativaById(ExperienciaEducativaDAOTest.java:58)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 10:16:35 WARN  ProblematicaAcademicaDAO:44 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Parameter index out of range (1 > number of parameters, which is 0).
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.ClientPreparedStatement.checkBounds(ClientPreparedStatement.java:1396)
	at com.mysql.cj.jdbc.ClientPreparedStatement.getCoreParameterIndex(ClientPreparedStatement.java:1409)
	at com.mysql.cj.jdbc.ClientPreparedStatement.setString(ClientPreparedStatement.java:1776)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.findProblematicaAcademicaByDescription(ProblematicaAcademicaDAO.java:25)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.findProblematicaAcademicaByDescription(ProblematicaAcademicaDAOTest.java:32)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 10:36:58 WARN  ProblematicaAcademicaDAO:91 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLSyntaxErrorException: Unknown column 'idProblematicaAcedemica' in 'where clause'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.deleteProblematicaAcademicaById(ProblematicaAcademicaDAO.java:84)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.deleteProblematicaAcademicaById(ProblematicaAcademicaDAOTest.java:53)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 10:38:40 INFO  ProblematicaAcademicaDAO:88 - Problematica academica eliminada con el id 2
2022-04-18 17:53:57 WARN  ProblematicaAcademicaDAO:79 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Problematica academica not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.findProblematicaAcademicaById(ProblematicaAcademicaDAO.java:63)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.findProblematicaAcademicaById(ProblematicaAcademicaDAOTest.java:62)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 18:06:28 WARN  ProblematicaAcademicaDAO:99 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLIntegrityConstraintViolationException: Cannot add or update a child row: a foreign key constraint fails
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:117)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.addProblematicaAcademica(ProblematicaAcademicaDAO.java:93)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.addProblematicaAcademica(ProblematicaAcademicaDAOTest.java:20)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 18:13:26 WARN  ProblematicaAcademicaDAO:79 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Problematica academica not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.findProblematicaAcademicaById(ProblematicaAcademicaDAO.java:63)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.findProblematicaAcademicaById(ProblematicaAcademicaDAOTest.java:62)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 18:17:44 WARN  ProblematicaAcademicaDAO:79 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Problematica academica not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.findProblematicaAcademicaById(ProblematicaAcademicaDAO.java:63)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.findProblematicaAcademicaById(ProblematicaAcademicaDAOTest.java:62)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-18 18:18:29 WARN  ProblematicaAcademicaDAO:79 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Problematica academica not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.findProblematicaAcademicaById(ProblematicaAcademicaDAO.java:63)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.findProblematicaAcademicaById(ProblematicaAcademicaDAOTest.java:62)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-21 20:36:32 WARN  PersonaDAO:30 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLSyntaxErrorException: Unknown column 'idsolucionaproblematicaacademica' in 'field list'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.SolucionAProblematicaAcademicaDAO.addSolucionProblematicaAcademica(SolucionAProblematicaAcademicaDAO.java:24)
	at test.uv.fei.tutorias/uv.fei.tutorias.SolucionAProblematicaacademicaDAOTest.addSolucionAProblematicaAcademica(SolucionAProblematicaacademicaDAOTest.java:14)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-21 21:51:28 WARN  PersonaDAO:82 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Column 'idsolucioaproblemticaacademica' not found.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:584)
	at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:851)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.SolucionAProblematicaAcademicaDAO.findSolucionAProblematicaAcademicaById(SolucionAProblematicaAcademicaDAO.java:73)
	at test.uv.fei.tutorias/uv.fei.tutorias.SolucionAProblematicaacademicaDAOTest.serchSolucionAProblematicaAcademica(SolucionAProblematicaacademicaDAOTest.java:26)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-21 21:52:54 WARN  PersonaDAO:82 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Column 'idsolucionaproblemticaacademica' not found.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.result.ResultSetImpl.findColumn(ResultSetImpl.java:584)
	at com.mysql.cj.jdbc.result.ResultSetImpl.getInt(ResultSetImpl.java:851)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.SolucionAProblematicaAcademicaDAO.findSolucionAProblematicaAcademicaById(SolucionAProblematicaAcademicaDAO.java:73)
	at test.uv.fei.tutorias/uv.fei.tutorias.SolucionAProblematicaacademicaDAOTest.serchSolucionAProblematicaAcademica(SolucionAProblematicaacademicaDAOTest.java:26)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-22 10:56:32 WARN  PersonaDAO:95 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Reporte de tutoria academica not found
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ReporteDeTutoriaAcademicaDAO.findReporteDeTutoriaById(ReporteDeTutoriaAcademicaDAO.java:76)
	at test.uv.fei.tutorias/uv.fei.tutorias.ReporteDeTutoriaDAOTest.findReporteDetutoriaById(ReporteDeTutoriaDAOTest.java:22)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-26 18:36:56 WARN  EstudianteProblematicaAcademicaDAO:57 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: ERROR: No se ha eliminado ninguna relacion con estudiante-problematicaAcademica
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO.deleteEstudinateProblematicaAcademicaById(EstudianteProblematicaAcademicaDAO.java:52)
	at test.uv.fei.tutorias/uv.fei.tutorias.EstudianteProblematicaAcademicaDAOTest.deleteEstudianteProblematicaAcademicaDAO(EstudianteProblematicaAcademicaDAOTest.java:20)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-28 21:06:28 WARN  ProblematicaAcademicaDAO:98 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLException: Field 'ProblematicaAcademica_ id_solucionProblematicaAcademica' doesn't have a default value
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.addProblematicaAcademica(ProblematicaAcademicaDAO.java:92)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.addProblematicaAcademica(ProblematicaAcademicaDAOTest.java:20)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-28 21:09:47 WARN  ProblematicaAcademicaDAO:98 - uv.fei.tutorias.bussinesslogic.PersonaDAO
java.sql.SQLSyntaxErrorException: Unknown column 'ProblematicaAcademica_' in 'field list'
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeInternal(ClientPreparedStatement.java:953)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1098)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdateInternal(ClientPreparedStatement.java:1046)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeLargeUpdate(ClientPreparedStatement.java:1371)
	at com.mysql.cj.jdbc.ClientPreparedStatement.executeUpdate(ClientPreparedStatement.java:1031)
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.ProblematicaAcademicaDAO.addProblematicaAcademica(ProblematicaAcademicaDAO.java:92)
	at test.uv.fei.tutorias/uv.fei.tutorias.ProblematicaAcademicaDAOTest.addProblematicaAcademica(ProblematicaAcademicaDAOTest.java:20)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-28 21:12:57 WARN  EstudianteProblematicaAcademicaDAO:48 - uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO
java.sql.SQLException: No se han encontrado estudiantes con la problematica academica numero: 20
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO.findEstudiantesProblematicaAcademicaById(EstudianteProblematicaAcademicaDAO.java:29)
	at test.uv.fei.tutorias/uv.fei.tutorias.EstudianteProblematicaAcademicaDAOTest.findestudiantesProblematicaAcademica(EstudianteProblematicaAcademicaDAOTest.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-28 21:32:11 WARN  EstudianteProblematicaAcademicaDAO:48 - uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO
java.sql.SQLException: No se han encontrado estudiantes con la problematica academica numero: 12
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO.findEstudiantesProblematicaAcademicaById(EstudianteProblematicaAcademicaDAO.java:29)
	at test.uv.fei.tutorias/uv.fei.tutorias.EstudianteProblematicaAcademicaDAOTest.findestudiantesProblematicaAcademica(EstudianteProblematicaAcademicaDAOTest.java:35)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
2022-04-28 21:46:09 WARN  EstudianteProblematicaAcademicaDAO:48 - uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO
java.sql.SQLException: No se han encontrado estudiantes con la problematica academica numero: 12
	at uv.fei.tutorias/uv.fei.tutorias.bussinesslogic.EstudianteProblematicaAcademicaDAO.findEstudiantesProblematicaAcademicaById(EstudianteProblematicaAcademicaDAO.java:29)
	at test.uv.fei.tutorias/uv.fei.tutorias.EstudianteProblematicaAcademicaDAOTest.findestudiantesProblematicaAcademica(EstudianteProblematicaAcademicaDAOTest.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at junit@4.13.2/org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at junit@4.13.2/org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at junit@4.13.2/org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at junit@4.13.2/org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at junit@4.13.2/org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at junit@4.13.2/org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at junit@4.13.2/org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at junit@4.13.2/org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at junit@4.13.2/org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at junit@4.13.2/org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at junit@4.13.2/org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
